src/Mixins/Expectation.php

```diff
@@ -1128,4 +1129,19 @@ public function toBeUuid(string $message = ''): self
 
         return $this;
     }
+
+    /**
+     * Asserts that the value is an anonymous class.
+     *
+     * @return self<TValue>
+     */
+    public function toBeAnonymous(string $message = ''): self
+    {
+        $this->toBeObject();
```

Comment:
Is this true? Can't I provide a string that is an anonymous class?

Reply:
@nunomaduro No, I think so

Reply:
Try to the get the class from the anonymous class like so `get_class($anonymousObject);` and use that string to check.

Reply:
@nunomaduro Sorry, I don't understand your mind

Reply:
I think what Nuno is meaning is that you can pass an anonymous class reference, for example:
```php
<?php

$anonymousClass = new class() {};

$class = get_class($anonymousClass);

var_dump((new ReflectionClass($class))->isAnonymous());
```

Reply:
but I don't understand what advantage of `get_class` at here